frontend/containers/DocumentListToolbar/DocumentListToolbar.jsx

```diff
@@ -215,11 +215,19 @@ class DocumentListToolbar extends Component {
     } = this.props
 
     if (bulkActionSelected === 'delete') {
-      actions.deleteDocuments({
-        api,
-        collection,
-        ids: state.documents.selected
-      })
+      if (collection == "mediaStore") {
```

Comment:
Can we use a strong comparison ( `===`)? Also, is `collection` not an object that may or may not have a `_media` property to identify it as a media collection? I'd rather not hardcode the bucket name here.

Reply:
Oops - missed that - I always do strong comparisons as a matter of habit and must have missed that.
Yep, just going to pass in the name of the bucket from above - in the above component we've passed a `currentCollection` string it turns out.

Reply:
Ah, yes. That changed as part of my refactoring. Do you mind updating the `propTypes` to reflect that?

Reply:
Actually it was an object after all in this context, it was made into one from a string.
Anyway I'll introduce an "isMedia" to avoid scanning for media collections and just let the view pass in true.